### DISCUSSION TITLE
Merge two simulation directories

### DISCUSSION THREAD

**Discussion by yiyuxingchen** [neperfepx/FEPX#87] [Q&A] [answered]
@mkasemer 
Hello, 
A problem arises when I try to merge two simulation directories into a single simulation directory.
`neper -S cycle1.sim,cycle2.sim -o cycle1-2.sim`

```
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Merging simulation directories...
Info   :   - Merging inputs...
Error  :   - Could not read field `**restart'.
Aborted
```

neper version 4.4.2-5
fepx version 1.3.1-1

[cycle.sim.zip](https://github.com/neperfepx/FEPX/files/8714270/cycle.sim.zip)

Thanks in advance.

**Comment by rquey** [comment, marked as answer]
Update Neper.
